docs/application/native/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,294 @@
+# Component Manager
+
+Component manager provides information about installed and running components.
+
+The main features of the Component Manager API include:
+
+- Managing component context
+
+  You can retrieve the running component context and operate on it. The `component_context_h` handle of Component Context API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__CONTEXT__MODULE.html) applications) is related to the running components and can be used to [manage the component context](#manage_context).
+
+- Managing component information
+
+  The `component_info_h` handle of Component Information API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__INFO__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__INFO__MODULE.html) applications) is related to the available components that are installed but not necessarily running.
+
+Iterator functions are used to travel through a list of components. `component_manager_foreach_component_context()` is used in running components and `component_manager_foreach_component_info()` is used in available components. Each function calls a callback function `component_manager_component_context_cb()` or `component_manager_component_info_cb()`, passing the handle for each component.
+
+## Prerequisites
+
+1. To use the Component Manager API (in [mobile](../../api/mobile/latest/group__CAPI__COMPONENT__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__COMPONENT__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
Request from @surbhig12 :

Global: Please remove the article "the" before Component Manager API. As in line number 9 and 13.

1. To use Component Manager API....

Reply:
Done